spec/models/user_spec.rb

```diff
@@ -6,11 +6,12 @@
     let(:guest_user) { create :user, email: "guest@user.com" }
     let(:admin_user) { create :user, email: "guest@user.com", admin: true}
 
-    it "the 'Admin' status is false" do
+    it "by standard, the 'Admin' status is false" do
       expect(guest_user.admin).to eq(false)
     end
 
-    it "the 'Admin' status is false" do
+    it "if 'Admin' status is set to true by another Admin,
```

Comment:
I think we could describe this test as : 'it "creates an admin user when admin: true is included in the create function" '